scrunch/views.py

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+from scrunch.helpers import shoji_entity_wrapper
+
+
+class DatasetViews:
+    def __init__(self, dataset_resource):
+        self.dataset_resource = dataset_resource
+
+    def create(self, name, columns):
```

Comment:
do u plan on adding `rows` at later time?

Reply:
and destination `project`?

Reply:
tried to keep it at a minimum this time, just to make sure we had support. I'm sure they'll request it and we can have smaller PRs for those.